frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -268,10 +272,11 @@ const PendingOrganizations = ({
                     <th scope="row">Contact information</th>
                     <th scope="row">Created</th>
                     <th scope="col">External ID</th>
-                    <th scope="col"></th>
+                    <th scope="col" aria-hidden></th>
+                    <th scope="col" aria-hidden></th>
                   </tr>
                 </thead>
-                <tbody>{orgRows()}</tbody>
+                <tbody aria-live={"polite"}>{orgRows()}</tbody>
```

Comment:
I added this attribute to follow the pattern for empty tables that we have implemented in [ManagePatients.tsx](https://github.com/CDCgov/prime-simplereport/blob/1868750e67424819fe366b4b214c69dd8adf0452/frontend/src/app/patients/ManagePatients.tsx#L172)